safeeyes/safeeyes/config/lang/tr.json

```diff
@@ -0,0 +1,46 @@
+{
+    "meta_info": {
+        "language_name": "Türkçe",
+        "language_name_en": "Turkish"
+    },
+    "exercises": {
```

Comment:
`exercises` structure is changed to:
`
"exercises": {
        "short_break_close_eyes": "Tightly close your eyes",
        "short_break_roll_eyes": "Roll your eyes",
        "short_break_rotate_clockwise": "Rotate your eyes in clockwise direction",
        "short_break_rotate_counter_clockwise": "Rotate your eyes in counterclockwise direction",
        "short_break_blink": "Blink your eyes",
        "short_break_focus_far_distance": "Focus on a point in the far distance",
        "short_break_drink_water": "Have some water",
        "long_break_walk": "Walk for a while",
        "long_break_lean_back": "Lean back at your seat and relax"
    }
`